packages/ui/tailwind-design-system.ts

```diff
@@ -428,48 +389,7 @@ export default {
       addUtilities(typographyStyles)
     }),
     plugin(({ addComponents }) => {
-      addComponents([
-        accordionStyles,
-        alertStyles,
-        badgeStyles,
-        buttonStyles,
-        dropdownStyles,
-        switchStyles,
-        labelStyles,
-        tagStyles,
-        linkStyles,
-        avatarStyles,
-        cardStyles,
-        cardSelectStyles,
-        paginationStyles,
-        progressStyles,
-        drawerStyles,
-        buttonLayoutStyles,
-        tabsStyles,
-        dialogStyles,
-        scrollAreaStyles,
-        popoverStyles,
-        tooltipStyles,
-        sidebarStyles,
-        breadcrumbStyles,
-        iconsAndLogos,
-        timeAgoCardStyles,
-        shortcutStyle,
-        toggleStyles,
-        toggleGroupStyles,
-        buttonGroupStyle,
-
-        // Form styles
-        selectStyles,
-        formSharedStyles,
-        checkboxStyles,
-        radioStyles,
-        textareaStyles,
-        inputStyles,
-        captionStyles,
-        multiSelectStyles,
-        tableV2Styles
-      ])
+      addComponents(ComponentStyles)
```

Comment:
Fixing to keep tailwind config clean and move component style imports to its relevant file.